src/utils/unitPluralizer.ts

```diff
@@ -0,0 +1,3 @@
+export const unitPluralizer = (value: number, unit: string): string => {
```

Comment:
Maybe `pluralizeUnit` is a better name? Pluralizer sounds more like an entity that does something after being initialized.